web_service/private_api/search_routes.js

```diff
@@ -1,46 +1,154 @@
 const expressUtils = require('shared/util/expressUtils');
-const { Client } = require('@elastic/elasticsearch');
 const u = require('shared/util/u');
+const { query } = require('shared/util/db.js');
 
 const searchRouter = (() => {
   const router = expressUtils.createRouter();
 
-  const client = new Client({ node: 'http://localhost:9200' });
+  router.get('/:query', async (req, res) => {
```

Comment:
query should be a query string?

Reply:
Updated to use a query param (`/search?q=mysearchterms`)